packages/DOS/common.dj

```diff
@@ -91,33 +104,59 @@ USE_ARES = 0
 #
 # Enable debug code in libcurl/curl
 #
-USE_DEBUG = 0
+USE_DEBUG = 1
```

Comment:
Personal preference

Reply:
It add a `-DDEBUGBUILD`. libcurl is unusable without that.